ehrql/tables/beta/core.py

```diff
@@ -117,23 +126,71 @@ class ons_deaths(PatientFrame):
             ),
         ],
     )
-    underlying_cause_of_death = Series(ICD10Code)
+    underlying_cause_of_death = Series(
+        ICD10Code,
+        description="Patient's underlying cause of death of death.",
+    )
     # TODO: Revisit this when we have support for multi-valued fields
-    cause_of_death_01 = Series(ICD10Code)
-    cause_of_death_02 = Series(ICD10Code)
-    cause_of_death_03 = Series(ICD10Code)
-    cause_of_death_04 = Series(ICD10Code)
-    cause_of_death_05 = Series(ICD10Code)
-    cause_of_death_06 = Series(ICD10Code)
-    cause_of_death_07 = Series(ICD10Code)
-    cause_of_death_08 = Series(ICD10Code)
-    cause_of_death_09 = Series(ICD10Code)
-    cause_of_death_10 = Series(ICD10Code)
-    cause_of_death_11 = Series(ICD10Code)
-    cause_of_death_12 = Series(ICD10Code)
-    cause_of_death_13 = Series(ICD10Code)
-    cause_of_death_14 = Series(ICD10Code)
-    cause_of_death_15 = Series(ICD10Code)
+    cause_of_death_01 = Series(
+        ICD10Code,
+        description="Medical condition mentioned on the death certificate.",
```

Comment:
Because the order of these codes is not meaningful I did not include the number (1-15) in the description.